src/specBuilder/chartSpecBuilder.ts

```diff
@@ -210,6 +211,14 @@ export const getDefaultSignals = (
 	];
 };
 
+export const getTimer = () => {
+	return {
+		name: 'timerValue',
+		value: '0',
+		on: [{ events: 'timer{16}', update: 'min(1, timerValue + (1 / 360))' }],
```

Comment:
Eventually we should use a template string for the numbers.  But for a starting point this works